### PR TITLE
Format function

### PR DESCRIPTION
**Pull request by StefanEnberg**

### CODE REVIEW

**Comment by StefanEnberg** [comment]
Closes #11

File: scripts/load-formating-functions [hunk 1]
```diff
@@ -0,0 +1,13 @@
+SET DateFormat='DD/MM/YYYY'; //This will be the default setting used by Date formatting functions. Read more about the default variables at https://www.qlikcore.com/services/qix-engine/script_reference/number_variables/
+
+LET InputDateFormat='YYYY-MM-DD';
+LET CustomDisplayDateFormat='YYYY-MM';
+
+Sales:
+    LOAD
+      //#Date() defines the input format of the date value
+      //Date() defines the that date value is a date and of what format it will us
```
Comment (hrigner) [inline scripts/load-formating-functions]: some typos - "the that" and "us"

**Review by hrigner — APPROVED** [review]
LGTM, just some minor typos

File: scripts/load-formating-functions [hunk 2]
```diff
@@ -0,0 +1,13 @@
+SET DateFormat='DD/MM/YYYY'; //This will be the default setting used by Date formatting functions. Read more about the default variables at https://www.qlikcore.com/services/qix-engine/script_reference/number_variables/
```
Comment (janmiderback) [inline scripts/load-formating-functions]: Maybe avoid line-ending comments? They are sometimes harder to read and harder to spot.

File: scripts/load-formating-functions [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+LET InputDateFormat='YYYY-MM-DD';
+LET CustomDisplayDateFormat='YYYY-MM';
+
+Sales:
+    LOAD
+      //#Date() defines the input format of the date value
```
Comment (janmiderback) [inline scripts/load-formating-functions]: Nit - Maybe empty space after `//` to keep comments consistent?

**Review by janmiderback — APPROVED** [review]
LGTM! Just two nit comments.

**Pull request merged**